File: Dataset/Leetcode/valid/62/125.cpp
 class Solution {
public:
    int XXX(int m, int n) {
        if(n<=1 || m<=1) return 1;
        int dp[1000] = {1};
        fill(dp,dp+m,1);
        while (--n>0){
            for (int i = 1; i < m; ++i) {
                dp[i] = dp[i]+dp[i-1];
            }
        }
        return dp[m-1];
    }
};

File: Dataset/Leetcode/train/118/125.py
 class Solution:
    def XXX(self, numRows: int) -> List[List[int]]:
        s = []
        temp = [1,1]
        for i in range(numRows):
            mi = [1]
            if i==0:
                mi = [1]
            elif i==1:
                mi.append(1)
            else:
                for j in range(1,i):
                    mi.append(temp[j-1]+temp[j])
                mi.append(1)    
            temp = mi
            s.append(mi)
        return s

